Source/Extensions/Blazorise.DataGrid/wwwroot/datagrid.js

```diff
@@ -1,18 +1,28 @@
 import { getRequiredElement } from "../Blazorise/utilities.js?v=1.5.2.0";
 
+const QUERYSELECTOR_ALL_COLUMNS = "tbody tr td";
+const QUERYSELECTOR_ALL_TABLE_HEAD_INPUT = "tbody tr td";
```

Comment:
Does this selector work for all providers?

Reply:
Actually I forgot to test. But I'd expect that they all obbey that basic structure for a table.

Reply:
Working on all providers.